tests/rpmdb.at

```diff
@@ -683,3 +683,20 @@ runroot rpm -q 'versiontest-1.0~2-1'
 [])
 
 RPMTEST_CLEANUP
+
+# ------------------------------
+AT_SETUP([rpmdb vacuum])
+AT_KEYWORDS([install rpmdb sqlite])
+cp -r /usr/lib/sysimage/rpm "${RPMTEST}"/tmp
+RPMTEST_CHECK([
+runroot rpm -D "_sqlite_vacuum 1024" -D "%_dbpath /tmp/rpm" -vv -U /data/RPMS/foo-1.0-1.noarch.rpm 2>&1 | grep VACUUM
+runroot rpm -D "_sqlite_vacuum 1024" -D "%_dbpath /tmp/rpm" -vv -e foo-1.0-1 2>&1 | grep VACUUM
+],
+[0],
+[D: VACUUM: 0
+D: Rpmdb Sqlite backend VACUUM maxfree: 1024, free: 397312 -> 0
```

Comment:
/usr/lib/sysimage/rpm is a "live" rpmdb, so these values can and will change, breaking the test.

Reply:
Yeah, that's kinda on purpose. I wanted a real database for testing. But may be the test suite can do without.